book/src/help_bn.md

```diff
@@ -603,6 +603,4 @@ Flags:
   -z, --zero-ports
           Sets all listening TCP/UDP ports to 0, allowing the OS to choose some
           arbitrary free ports.
-```
-
-<style> .content main {max-width:88%;} </style>
+```n<style> .content main {max-width:88%;} </style>
```

Comment:
this looks wrong, did the script do this?

Reply:
which OS did you run it on?

Reply:
ran a make cli-local

Reply:
mac os

Reply:
yeah I thought something was off but the script/cli.sh had this

<img width="832" alt="Screenshot 2024-09-26 at 12 10 11 AM" src="https://github.com/user-attachments/assets/e34ee300-95e2-44d9-974e-446b86739781">

Reply:
ah yeah, macOS sed is a bit different

I'll run it on Linux and fix this

Reply:
should I add this 

```
if [[ "$OSTYPE" == "darwin"* ]]; then
    sed -i '' -e '$a\'$'\n''<style> .content main {max-width:88%;} </style>' "$file"
else
    sed -i -e '$a\'$'\n''<style> .content main {max-width:88%;} </style>' "$file"
fi
```

to make our lives easier in the future?

Reply:
that would be great actually

Reply:
thank you

Reply:
that regexp might need a tweak? it looks the same as the Linux one?

Reply:
well today I learned I'm still a noob programmer, but yeah I was defeated by a linux command.  
I think using a `printf` just like the command to add the code block quote right above this line just solves the problem in an easy way and sed is just a little ambiguous for different distros. 

there's another issue I'd like to point out 

<img width="1120" alt="Screenshot 2024-09-26 at 9 39 40 AM" src="https://github.com/user-attachments/assets/30cdfbb4-0ba7-4148-b983-03ba5460d9ac">

when I run `make cli` which uses docker this happens, if that would have worked I would have not come across the current issue. do you want to create a separate tracking issue for that?

Reply:
Hmm, I don't think that image is ever going to be public. Maybe that's just a doc issue

Reply:
Oh, it is public but it doesn't work on ARM.

Reply:
If you have Rosetta you can run x86_64 images on macOS using emulation. It's slower but it works.